ResourceManager/GitExtensionsControlInitialiser.cs

```diff
@@ -27,11 +27,11 @@ public GitExtensionsControlInitialiser(GitExtensionsControl control)
             _translate = control;
         }
 
-        private void LoadHandler(object control, EventArgs e)
+        private void LoadHandler(object? sender, EventArgs e)
         {
             if (!_initialiseCompleteCalled && !IsDesignModeActive)
             {
-                throw new Exception($"{control.GetType().Name} must call {nameof(InitializeComplete)} in its constructor, ideally as the final statement.");
+                throw new Exception($"{((Control)sender!).GetType().Name} must call {nameof(InitializeComplete)} in its constructor, ideally as the final statement.");
```

Comment:
This looks wrong. The method advertises it accepts null but then throws if it receives it. Either change the signature and suppress usages of the method, or handle null more gracefully here.